code/modules/awaymissions/gateway.dm

```diff
@@ -171,6 +171,8 @@ GLOBAL_LIST_EMPTY(gateway_destinations)
 	var/obj/effect/gateway_portal_bumper/portal
 	/// Visual object for handling the viscontents
 	var/obj/effect/gateway_portal_effect/portal_visuals
+	/// Overlay of the lights. They light up fully when it charges fully.
+	var/mutable_appearance/light_overlay
```

Comment:
Shouldn't you QDEL_NULL this on Destroy?

Reply:
no need